wallet/package.json

```diff
@@ -76,6 +76,7 @@
     },
     "dependencies": {
         "@mysten/sui.js": "workspace:*",
+        "@popperjs/core": "^2.11.6",
```

Comment:
Popper has a successor called Floating UI, which I'd use if you can instead: https://floating-ui.com/

Reply:
Oh cool looks fancy! I am not familiar and I have limited time today, do you think changing it in another PR would be ok? (I will create an issue to track it)